sys/kern/src/arch/riscv32.rs

```diff
@@ -51,6 +52,7 @@ macro_rules! uassert {
     };
 }
 
+#[allow(unused)]
 macro_rules! uassert_eq {
```

Comment:
There is nothing in the entire repo that refers to uassert_eq. Should we drop it completely? arm also doesn't provide it.

Reply:
I didn't realize that. Yeah, might as well cut it out then.